app_dart/lib/src/service/config.dart

```diff
@@ -489,3 +495,27 @@ interface class Config {
     return GithubService(github);
   }
 }
+
+/// Flags for the service that can be updated dynamically with out a restart.
+///
+/// Should be read from git/HEAD/app_dart/config.yaml and cached between
+/// services.
+@JsonSerializable()
+class DynamicConfig {
+  /// Upper limit of commit rows to be backfilled in API call.
+  ///
+  /// This limits the number of commits to be checked to backfill. When bots
+  /// are idle, we hope to scan as many commit rows as possible.
+  @JsonKey(defaultValue: 50)
+  final int backfillerCommitLimit;
+
+  DynamicConfig({required this.backfillerCommitLimit});
+
+  /// Connect the generated [_$DynamicConfigFromJson] function to the `fromJson`
+  /// factory.
+  factory DynamicConfig.fromJson(Map<String, dynamic> json) =>
```

Comment:
nit: Use `Map<String, Object?>` instead of `Map<String, dynamic>`.

It's the same backend type representation, but avoids accidental dynamic access.

Reply:
Done